Documentation/index.rst

```diff
@@ -38,6 +38,9 @@ The documentation is divided into the following sections:
 * :ref:`dev_guide` : Gives background to those looking to develop and contribute
   modifications to the Cilium code or documentation.
 
+`Hands-on tutorial <https://play.instruqt.com/isovalent/invite/j4maqox5r1h5>`_ 
+  in a live environment to quickly get started with Cilium.
```

Comment:
Given that this is outside of the list now, it would read better if the note was a full sentence. How about something like:

```
A `hands-on tutorial <...>`__ in a live environment is also available
for users looking for a way to quickly get started and experiment
with Cilium.`
```

What do you think?

Reply:
It looks good to me 💯